### PR TITLE
[AC-420] Fill up 'Glossary' page

### PR DESCRIPTION
**Pull request by f4ww4z**
https://issues.openmrs.org/browse/AC-420

Added some key terms and their definitions.

### CODE REVIEW

File: glossary.md [hunk 1]
```diff
@@ -1,7 +1,55 @@
...
+
+**metadata**: A piece of information that describes other information.
+
+**module**: A software package that extends OpenMRS functionality in specific ways; often developed by others in the OpenMRS community.
+
+**observation**: One piece of information that is recorded about a person at a moment in time.
+
+**password**: String of characters used for user authentication to prove identity or access approval to gain access to a resource, which is to be kept secret from those not allowed access.
+
+**patient identifier**: Any unique number that can identify a patient. Examples are a Medical Record Number, a National ID, a Social Security Number, a driver's license number, etc.
```
Comment (Sub6Resources) [inline glossary.md]: In alphabetical order, patient should come before patient identifier

File: glossary.md [hunk 2]
```diff
@@ -1,7 +1,55 @@
...
+
+**patient**: A person receiving health care services.
+
+**person**: Every individual who is referred to in any patient's record in OpenMRS must be stored in the system.
+
+**provider**: A person or company that provides a health care service to you. E.g. doctors, nurses, clinicans or physicians.
+
+**session location**: A provider's current location when inputting data in an EMR encounter. E.g. Outpatient clinic, laboratory.
```
Comment (Sub6Resources) [inline glossary.md]: Same as above but with session and session location

File: glossary.md [hunk 3]
```diff
@@ -1,7 +1,55 @@
...
+
+**session**: The period during a provider's interaction with an EMR system.
+
+**user**: A person who uses OpenMRS, or more specifically the data in the system representing that person.
+
+**username**: An identification used by a person with access to a computer, network, or online service.
+
+**visit note**: A quick note on experiences regarding a patient's visit.
```
Comment (Sub6Resources) [inline glossary.md]: Same as above but with visit and visit note.

File: glossary.md [hunk 4]
```diff
@@ -1,7 +1,55 @@
...
+
+**clinician**: A doctor, nurse, or other clinical officer who provides health care to patients.
+
+**concept**: The idea that encompasses any question which can be asked about a patient; an observable point of data.
+
+**data**: A piece of knowledge that can be reduced to a single value.
+
+**electronic medical record (EMR)**: A computer system that allows for recording, storage, and retrieval of information related to the delivery of health care to patients.
+
+**encounter**: A clinical transaction between a patient and one or more healthcare providers for the purpose of providing patient services or assessing the health status of the patient. An encounter happens at one point in time. [http://www.astm.org/Standards/E1384.htm](http://www.astm.org/Standards/E1384.htm)
```
Comment (Sub6Resources) [inline glossary.md]: What is the purpose of the link in this definition?

**Review by Sub6Resources — COMMENTED** [review]
Just some small things I noticed as I reviewed your PR.

File: glossary.md (hunk 4 above)
Comment (f4ww4z) [reply glossary.md]: To give a more elaborate description on what  an encounter is. I notice now it is withdrawn in March 2017. I'll remove this link.

File: glossary.md (hunk 1 above)
Comment (f4ww4z) [reply glossary.md]: Updating

File: glossary.md (hunk 2 above)
Comment (f4ww4z) [reply glossary.md]: Updating now

File: glossary.md (hunk 3 above)
Comment (f4ww4z) [reply glossary.md]: Updating now

**Review by BartlomiejRasztabiga — APPROVED** [review]
Looks good to me

**Review by Sub6Resources — APPROVED** [review]
Looks great! 👍

**Comment by shivtej1505** [comment]
Thanks, @f4ww4z. Merging now!

**Pull request merged**